test/integrations/destinations/af/router/data.js

```diff
@@ -0,0 +1,398 @@
+const data = [
+    {
+      "name": "af",
+      "description": "Test 0",
```

Comment:
I would suggest using a better description for the test. Without it, people will have a hard time maintaining them, especially given the verbose input.